src/webhooks/options-automator/options-automator.ts

```diff
@@ -1,21 +1,41 @@
+import { v1 } from '@datadog/datadog-api-client';
 import * as Sentry from '@sentry/node';
 import { KnownBlock } from '@slack/types';
 import { FastifyRequest } from 'fastify';
+import moment from 'moment-timezone';
 
 import { OptionsAutomatorResponse } from '@types';
 
 import { bolt } from '@/api/slack';
 import * as slackblocks from '@/blocks/slackBlocks';
-import { FEED_OPTIONS_AUTOMATOR_CHANNEL_ID } from '@/config';
+import {
+  DATADOG_API_INSTANCE,
+  FEED_OPTIONS_AUTOMATOR_CHANNEL_ID,
+} from '@/config';
 
 export async function handler(
   request: FastifyRequest<{ Body: OptionsAutomatorResponse }>
 ) {
   const { body }: { body: OptionsAutomatorResponse } = request;
   await messageSlack(body);
+  await sendOptionAutomatorUpdatesToDataDog(body);
   return {};
 }
 
+export async function sendOptionAutomatorUpdatesToDataDog(
+  message: OptionsAutomatorResponse
+) {
+  // Not sure how detailed we want the text message to be, but this should be fine initially
+  const params: v1.EventCreateRequest = {
+    title: 'Options Automator Update',
+    text: JSON.stringify(message),
+    alertType: 'info',
+    dateHappened: moment.now(),
+    tags: [message.region],
```

Comment:
@hubertdeng123 Not sure how you do string interpolation in Typescript, but could you tag this like: `["sentry_region:${message.region}"]

Reply:
yep, can do this